astropy/visualization/wcsaxes/coordinate_helpers.py

```diff
@@ -293,6 +293,12 @@ def set_format_unit(self, unit, decimal=None, show_decimal_unit=True):
         self._formatter_locator.decimal = decimal
         self._formatter_locator.show_decimal_unit = show_decimal_unit
 
+    def get_format_unit(self):
```

Comment:
Can you add a test? (well just add a line to a test that does ``set_format_unit``

Reply:
+1